### ISSUE TITLE
Crash with UnicodeEncodeError

### ISSUE THREAD

**Issue by jensgreen** [walles/px#41]
```
~ ❯❯❯ ptop

Traceback (most recent call last):
  File "/usr/local/bin/ptop", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/px/px.py", line 79, in main
    px_top.top()
  File "/usr/local/lib/python2.7/site-packages/px/px_top.py", line 252, in top
    _top()
  File "/usr/local/lib/python2.7/site-packages/px/px_top.py", line 216, in _top
    redraw(load_bar, baseline, rows, columns)
  File "/usr/local/lib/python2.7/site-packages/px/px_top.py", line 181, in redraw
    lines = get_screen_lines(load_bar, baseline, rows, columns)
  File "/usr/local/lib/python2.7/site-packages/px/px_top.py", line 162, in get_screen_lines
    toplist_table_lines = px_terminal.to_screen_lines(get_toplist(baseline), columns)
  File "/usr/local/lib/python2.7/site-packages/px/px_terminal.py", line 84, in to_screen_lines
    proc.cmdline)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 946-956: ordinal not in range(128)
```

```
~ ❯❯❯ px                                                                                                                                           ✘ 1
Traceback (most recent call last):
  File "/usr/local/bin/px", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/px/px.py", line 101, in main
    lines = px_terminal.to_screen_lines(px_process.order_best_last(procs), columns)
  File "/usr/local/lib/python2.7/site-packages/px/px_terminal.py", line 84, in to_screen_lines
    proc.cmdline)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 946-956: ordinal not in range(128)
```

**Comment by walles** [comment]
Fixed by b828d6f

**Comment by jensgreen** [comment]
Fixed!
